build.gradle

```diff
@@ -76,7 +76,11 @@ dependencies {
 
     //Zooming Android ImageView.
     implementation 'com.github.chrisbanes:PhotoView:2.0.0'
+
     implementation 'com.gitlab.ownclouders:dav4android:oc_support'
+
+    // Convert Java Objects into JSON and back
+    implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
I'd recommend you using Moshi instead

Reply:
This looked pretty easy to use, what are the advantages of Moshi?